dataland-frontend/src/services/ApiClients.ts

```diff
@@ -1,30 +1,99 @@
 import { Configuration } from "@clients/backend/configuration";
-import {
-  CompanyDataControllerApi,
-  type CompanyDataControllerApiInterface,
-  MetaDataControllerApi,
-  type MetaDataControllerApiInterface,
-} from "@clients/backend/api";
 import { DocumentControllerApi } from "@clients/documentmanager";
 import { QaControllerApi } from "@clients/qaservice";
 import type Keycloak from "keycloak-js";
 import { ApiKeyControllerApi, type ApiKeyControllerApiInterface } from "@clients/apikeymanager";
 import { RequestControllerApi, type RequestControllerApiInterface } from "@clients/communitymanager";
+import axios, { type AxiosInstance } from "axios";
 import { updateTokenAndItsExpiryTimestampAndStoreBoth } from "@/utils/SessionTimeoutUtils";
 import { type FrameworkDataTypes } from "@/utils/api/FrameworkDataTypes";
 import { type FrameworkDataApi } from "@/utils/api/UnifiedFrameworkDataApi";
 import { getUnifiedFrameworkDataControllerFromConfiguration } from "@/utils/api/FrameworkApiClient";
+import * as backendApis from "@clients/backend/api";
+
+interface ApiBackendClients {
+  actuator: backendApis.ActuatorApiInterface;
+  companyDataController: backendApis.CompanyDataControllerApiInterface;
+  metaDataController: backendApis.MetaDataControllerApiInterface;
+  euTaxonomyDataForFinancialsController: backendApis.EuTaxonomyDataForFinancialsControllerApiInterface;
```

Comment:
The framework-specific API Controllers should all be handled through the `getUnifiedFrameworkDataController` function. Therefore, they can be removed here (and below)

Reply:
Removed.

Reply:
ok